src/_macosx.m

```diff
@@ -26,13 +23,6 @@
 #define COMPILING_FOR_10_10
 #endif
 
-/* Use Atsui for Mac OS X 10.4, CoreText for Mac OS X 10.5 */
-#ifndef COMPILING_FOR_10_5
```

Comment:
Does this mean we no longer support 10.4?  Looks like 10.4 came out in 2005 and was last updated in 2007 and is unsupported by apple as of 2009 (all according to wikipedia https://en.wikipedia.org/wiki/Mac_OS_X_Tiger) so I think this is OK.